en/user_guide/user.rst

```diff
@@ -0,0 +1,48 @@
+User, Role and Group
+====================
+
+Open Orchestra offers the possibility to manage different users with different rights.
+
+User Management
+---------------
+
+The users page allows to search, create, edit and delete users.
+
+At the creation, some fields are required :
+
+* **first Name**
+* **last Name**
+* **e-mail**
+* **username**
+* **password**
+
+Once the user is created, you can add some information regarding his default language and his group.
+
+Role Management
+---------------
+
+A role gives one type of authorization for a user.
+
+Roles contains:
+
+* **name**
+* **description for each site languages**:  to display in admin panel
+* **original/destination status**: optional parameters to sharp roles for specifics content types and define an edition line.
+
+Some roles are already creating. By example:
+
+* **ROLE_ACCESS_USER**:  allows to manage users
+* **ROLE_ACCESS_TREE_TEMPLATE**: allows to manage templates
+* **ROLE_ACCESS_KEYWORD**: allows to manage keywords
+* **ROLE_ACCESS_LOG**: allows to see logs
+* **ROLE_ACCESS_REDIRECTION**: allows to manage redirections
+* **ROLE_ACCESS_ROLE**: allows to manage roles
+* **ROLE_ACCESS_SITE**: allows to manage websites
+* **ROLE_ACCESS_STATUS**: allows to manage status
+
+Group Management
+----------------
+
+Each group has a name, a list of roles and is associated to only one website.
+Users in a group have some rights and access for website define by group's roles.
```

Comment:
Maybe something like "Users in a group inherit rights defined in this group for the website associated"